lib/TPP/GPU/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ add_mlir_library(TPPGPU
     MLIRGPUToNVVMTransforms
 )
 
-if (NOT TPP_GPU STREQUAL "")
+if (TPP_GPU MATCHES "cuda")
```

Comment:
Weird that the definition is generic `TPP_GPU_ENABLE`... Perhaps that could also be Cuda specific?

Reply:
I'll have to probably reorganize it later anyway but sure why not.